### Debug Output No `pybind`

```
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Code\<user>\minimal_examples\HelloSayer\x64\Debug\HelloSayerLibCppTest.exe'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\ntdll.dll'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\kernel32.dll'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\KernelBase.dll'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\msvcp140d.dll'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\vcruntime140d.dll'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\vcruntime140_1d.dll'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\ucrtbased.dll'. Symbols loaded.
The thread 0x384c has exited with code 0 (0x0).
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Code\<user>\minimal_examples\HelloSayer\x64\Debug\HelloSayerLib.dll'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Unloaded 'C:\Code\<user>\minimal_examples\HelloSayer\x64\Debug\HelloSayerLib.dll'
Exception thrown at 0x00007FFDAA65A719 in HelloSayerLibCppTest.exe: Microsoft C++ exception: boost::wrapexcept<boost::system::system_error> at memory location 0x0000003252D5E430.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\kernel.appcore.dll'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\msvcrt.dll'. Symbols loaded.
'HelloSayerLibCppTest.exe' (Win32): Loaded 'C:\Windows\System32\rpcrt4.dll'. Symbols loaded.
The thread 0x1e3c has exited with code 1 (0x1).
The thread 0x51a4 has exited with code 1 (0x1).
The program '[7516] HelloSayerLibCppTest.exe' has exited with code 1 (0x1).
```